components/swatch/stories/swatch.stories.js

```diff
@@ -49,23 +57,193 @@ export default {
 			control: "boolean",
 		},
 		isSelected,
+		borderStyle: {
+			name: "Border style",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string", required: true },
+				category: "Component",
+			},
+			options: ["default", "noBorder", "lightBorder"],
+			control: "select",
+		},
+		isRectangle: {
+			name: "Rectangle",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+		},
+		imageUrl: {
+			name: "Image url",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "file", accept: ".svg,.png,.jpg,.jpeg,.webc" },
+		},
+		gradient: {
+			name: "Gradient values",
+			description: "Input an example gradient.",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			control: "color",
+		},
+		isMixedValue: {
+			name: "Mixed value",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+		},
 	},
 	args: {
 		rootClass: "spectrum-Swatch",
 		size: "m",
 		isSelected: false,
 		isDisabled: false,
 		rounding: "regular",
-		swatchColor: "rgb(174, 216, 230)"
+		swatchColor: "rgb(174, 216, 230)",
+		borderStyle: "default",
+		isRectangle: false,
+		isMixedValue: false,
 	},
 	parameters: {
 		componentVersion: version,
 	},
 };
 
+/**
+ * The medium size is the default and most frequently used option. By default, a swatch has a square shape.
+ */
 export const Default = SwatchGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+/**
+ * Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+ */
+export const Sizing = (args, context) => Sizes({
```

Comment:
I'm noticing for swatch, that the docs site shows a lot of options with less than 100% opacity (including in the docs site's sizing variant), meaning we see the checkerboard background. I can get that to show up on Storybook in the Default story (and in the testing preview), but wondering if leaving them out of the Docs page was a deliberate choice? It might be a nice thing for a user to see there in a story or two.

Reply:
It wasn't particularly deliberate. I was concerned I was already adding too many weird stories with multiple variants contained in one template group, so I opted for just using what was already there with the opaque swatch and the default swatch color.

But I agree- I like having the opacity visual. I added it to the rectangle, sizing, rounding and gradient stories. https://github.com/adobe/spectrum-css/pull/2925/commits/280706d0bd79f5eb466590baf95cc414ebe25b31